config/default_multi_subset.yaml

```
@@ -0,0 +1,10 @@
+---
+subsetName: default subset
+description: basic test
+configuration:
+    - ref: default
+      test: default
+    - ref: default 
+      test: bcstc
+    - ref: newProcessors
+      test: bcstc
```

To be changed to
```suggestion
    - ref: bcstc 
      test: bcstc
    - ref: newProcessors
      test: newProcessors
```
in order to have the same ref and test configurations in each case

---

Done.